### DISCUSSION TITLE
A page with multiple `tabs` blocks should automatically change the selected tab in all `tabs` blocks if title is the same.

### DISCUSSION THREAD

**Discussion by electricbolt** [GitbookIO/community#662] [Feature Requests]
This page:

https://docs.electricbolt.co.nz/configuration-subclasses/supported-property-types/integer

has multiple tabs blocks down the page. Each block has the following: 'Objective-C', 'Swift', 'Java', 'Kotlin', 'Dart'. Selecting one tab, e.g. 'Swift' should automatically select all other tabs blocks to the 'Swift' tab.

- The GitBook editor, **when not editing**, exhibits this desired behaviour. 
- The GitBook editor, when editing does not exhibit this desired behaviour.
- The rendered docs on my website does not exhibit this desired behaviour.

<img width="835" alt="Screenshot 2024-07-24 at 1 00 50 PM" src="https://github.com/user-attachments/assets/c647431a-2c1c-413a-8de0-6928a6d5a080">

**Comment by aikoii** [comment]
Hi @electricbolt! Thanks for your suggestion. We have this in our backlog and will keep you posted 🙂.

**Comment by aebranton** [comment]
Seconded for this. Right now trying to give someone examples in various languages is a nightmare.
